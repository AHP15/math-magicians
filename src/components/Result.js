import React from 'react';
import PropTypes from 'prop-types';

const Result = ({ result }) => (
  <div className="result">
    <p>{result}</p>
  </div>
);

Result.propTypes = {
  result: PropTypes.string.isRequired,
};
export default Result;
