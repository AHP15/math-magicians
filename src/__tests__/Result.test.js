import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Result from '../components/Result';

it('renders Result correctly', () => {
  const tree = renderer
    .create(<BrowserRouter><Result /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
