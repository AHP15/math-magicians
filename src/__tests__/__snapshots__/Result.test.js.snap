// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Result correctly 1`] = `
<div
  className="result"
>
  <p />
</div>
`;
